addons/60fps/addon.json

```diff
@@ -1,10 +1,10 @@
 {
-  "name": "60FPS project player mode",
-  "description": "Alt+Click the green flag to toggle 60FPS.",
+  "name": "Customizable project framerate",
+  "description": "Lets you run projects at higher framerates so they look smoother. Hold Alt and click the green flag to toggle between Scratch's default 30 FPS and a custom framerate set below.",
```

Comment:
I'm not sure how important this is, but previously `Alt+Click` was part of the addon name, then it became part of the first sentence of the description, and now it was moved to the second. After each of these changes there's probably more people that don't know how to toggle this

Reply:
Maybe changing the title to "Alt+Click green flag to increase FPS" might do the job, like "Ctrl+Enter to post".

Reply:
> Maybe changing the title to "Alt+Click green flag to increase FPS" might do the job, like "Ctrl+Enter to post".

The issue is we prefer noun phrases.

Reply:
> The issue is we prefer noun phrases.

"Do not run duplicated blocks"

Reply:
> > The issue is we prefer noun phrases.
> 
> "Do not run duplicated blocks"

It's not a strict rule but it is in the best practices.
> Use nouns instead of verbs (for example, `Customizable new sprite position` instead of `Change new sprite position`) unless doing so would result in the addon name being overly complicated.

I think using a noun phrase here works better and may be shorter as well. It's not exactly the same as "Ctrl+Enter to post".

Reply:
OK that's true. But how should we get around that problem then?

Reply:
"Alt+Click green flag for higher FPS"

Reply:
I would try to keep `Alt+Click the green flag` within the first 10 words of the description

Reply:
"Higher FPS by Alt+Click on green flag", or something to that avail might work.